mod/notifications/views/default/notifications/subscriptions/collections.php

```diff
@@ -7,19 +7,17 @@
 $user = $vars['user'];
 
 //@todo JS 1.8: no ?>
-<script type="text/javascript">
-	
+<script>
 	function setCollection(members, method, id) {
```

Comment:
I don't see why it needs to be. We know global functions are bad, but converting these to global vars with function expressions wouldn't buy us anything AFAICT.